test/controllers/secure.register.filter.controller.spec.ts

```diff
@@ -98,21 +114,63 @@ describe( "SECURE REGISTER FILTER controller", () => {
   });
 
   describe("POST tests", () => {
-    test(`renders the ${config.USE_PAPER_PAGE} page when yes is selected`, async () => {
+    test(`renders the ${config.USE_PAPER_PAGE} page when yes is selected and REDIS_removal flag is set to OFF`, async () => {
+      mockIsActiveFeature.mockReturnValueOnce(false);
+      mockGetApplicationData.mockReturnValueOnce({});
+      mockIsActiveFeature.mockReturnValueOnce(false);
```

Comment:
can you group the mockIsActiveFeature(s) so they are on adjacent lines

Reply:
The order of the mockIsActiveFeature(s) is deliberate: it follows the calling pattern in the code. But happy to group them by name if you think it a better approach.

Reply:
it's ok , i thought it might be easier to change them in future but it's only a minor comment. Happy to leave as is.